schemas/bookbrainz.sql

```diff
@@ -566,6 +566,36 @@ ALTER TABLE bookbrainz.language_set__language ADD FOREIGN KEY (language_id) REFE
 ALTER TABLE bookbrainz.edition_data ADD FOREIGN KEY (language_set_id) REFERENCES bookbrainz.language_set (id);
 ALTER TABLE bookbrainz.work_data ADD FOREIGN KEY (language_set_id) REFERENCES bookbrainz.language_set (id);
 
+CREATE TABLE bookbrainz.title_type (
+	id SERIAL PRIMARY KEY,
+	title VARCHAR(40) NOT NULL CHECK (title <> ''),
```

Comment:
Hmm, is 40 characters perhaps a bit limiting? I'd perhaps go with `TEXT NOT NULL UNIQUE CHECK (title <> '')` (like https://github.com/bookbrainz/bookbrainz-sql/blob/master/schemas/bookbrainz.sql#L305)